web/src/components/GridCell.js

```diff
@@ -83,9 +84,9 @@ class SingleGridCell extends React.Component {
               </Typography>
               <StatusChip status={this.props.SingleGridCellData.status} />
               <div>
-                <IconButton>
+                <Icon>
```

Comment:
The alignment is awkward but we are probably going to change it later when we add in the time. LGTM.